### PR TITLE
kubectl_test.go Remove use of NewRandomVMI func

### PR DESCRIPTION
**Pull request by Barakmor1**
NewRandomVMI is deprecated, libvmi should be used instead.

### What this PR does
Before this PR:

After this PR:

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/kubectl_test.go [hunk 1]
```diff
@@ -105,7 +105,10 @@ var _ = Describe("[sig-compute]oc/kubectl integration", decorators.SigCompute, f
 		BeforeEach(func() {
 			virtCli = kubevirt.Client()
 
-			vm = libvmi.NewVirtualMachine(tests.NewRandomVMI())
+			vm = libvmi.NewVirtualMachine(libvmifact.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
```
[the remark this answers is not in the document]
Comment (Barakmor1) [reply tests/kubectl_test.go:110]: I think that they are need to pause the vmi.

[the remark this answers is not in the document]
Comment (fossedihelm) [reply tests/kubectl_test.go:110]: Ok, it is because we want to make the VM live-migratable.
/lgtm

**Review by fossedihelm — APPROVED** [review]

**Review by EdDev — APPROVED** [review]

**Comment by iholder101** [comment]
Thanks @Barakmor1!

/approve

**Pull request merged**